portal/templates/contract/components/loan-transactions-table.njk

```diff
@@ -29,9 +29,10 @@
           <tbody class="govuk-table__body">
 
             {% for loan in loans %}
+            {% set loanLinkParams = { params: params, loan: loan } %}
```

Comment:
Should they be different variable names?

Reply:
updated